fiducial_vlam/CMakeLists.txt

```diff
@@ -6,22 +6,22 @@ project(fiducial_vlam)
 #=============
 
 # Default to C99
-if(NOT CMAKE_C_STANDARD)
+if (NOT CMAKE_C_STANDARD)
   set(CMAKE_C_STANDARD 99)
-endif()
+endif ()
 
 # Default to C++14
-if(NOT CMAKE_CXX_STANDARD)
+if (NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 14)
-endif()
+endif ()
 
 # Emulate colcon
-if($ENV{CLION_IDE})
+if ($ENV{CLION_IDE})
   message("Running inside CLion")
```

Comment:
I've learned that message("foo") will go to stderr, while message(STATUS "foo") will go to stdout, which works better when running in colcon. Doesn't matter in this case, but just FYI.

Reply:
Good to know!